addons/tmf_loadouts/loadouts/US_Rangers_2020.hpp

```diff
@@ -0,0 +1,657 @@
+	sidearmAttachments[] = {
+		"hlc_acc_tlr1",
+		"optic_mrd_black"
+	};
+	magazines[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_21Rnd_9x19_M17_Coyote")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	backpackItems[] = {
+		LIST_3("hlc_200rnd_556x45_M_SAW")
+	};
+	items[] += {"ACE_EntrenchingTool"};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class rat : r {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_M136"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	vest[] = {"CUP_V_CPC_lightbelt_mc"};
+	primaryWeapon[] = {
+		"CUP_srifle_M110",
+		"CUP_srifle_m110_kac"
+	};
+	scope[] = {
+		"cup_optic_leupoldmk4",
+		"cup_optic_leupoldmk4_10x40_lrt_desert"
+	};
+	attachment[] = {"cup_acc_anpeq_15"};
+	silencer[] = {"cup_muzzle_snds_m110"};
+	bipod[] = {"cup_bipod_vltor_modpod"};
+	sidearmWeapon[] = {"CUP_hgun_M17_Coyote"};
+	sidearmAttachments[] = {
+		"hlc_acc_tlr1",
+		"optic_mrd_black"
+	};
+	magazines[] = {
+		LIST_6("CUP_20Rnd_762x51_B_M110"),
+		LIST_3("CUP_21Rnd_9x19_M17_Coyote"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+	backpackItems[] = {
+		"ACE_Kestrel4500",
+		"ACE_MapTools",
+		"ACE_ATragMX",
+		"ACE_SpottingScope",
+		"ACE_Tripod"
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_M240"};
+	scope[] = {"cup_optic_elcan_specterdr_black"};
+	backPack[] = {"CFP_Kitbag_MCam_Grn"};
+	magazines[] = {
+		"hlc_200Rnd_762x51_M_M60E4",
+		LIST_3("hlc_100Rnd_762x51_Barrier_M60E4"),
+		LIST_3("CUP_21Rnd_9x19_M17_Coyote")
+	};
+};
+
+class mmgac : aar {
+	displayName = "MMG Ammo Carrier";
+	backPack[] = {"CFP_Kitbag_MCam_Grn"};
+	backpackItems[] = {
+		LIST_4("hlc_100Rnd_762x51_Barrier_M60E4")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	backPack[] = {"CFP_Kitbag_MCam_Grn"};
+	backpackItems[] = {
+		LIST_4("hlc_100Rnd_762x51_Barrier_M60E4")
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Yardage450",
+		"ItemRadio"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"ace_csw_staticM2ShieldCarry"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripod"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"B_Carryall_mcamo"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripodLow"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CFP_Kitbag_MCam_Grn"};
+	secondaryWeapon[] = {"CUP_launch_MAAWS"};

```

Comment:
Considering it is 2020, the vanilla MAAWS would probably fit nicely. https://en.wikipedia.org/wiki/Carl_Gustaf_recoilless_rifle#2014:_M4_(M3E1_in_USA)